### DISCUSSION TITLE
shortcode wiki

### DISCUSSION THREAD

**Discussion by BunnySakura** [adityatelange/hugo-PaperMod#1485] [Q&A]
我注意到PaperMod提供了一些`shortcode`，但是并未找到对应的说明，能否介绍一下作用？谢谢！
I noticed that PaperMod provides some `shortcode`, but I didn't find the corresponding instructions. Can you introduce the function? Thanks!

**Comment by mshafat** [comment]
Bump...would love to know more about this from the dev...
